tsconfig.json

```diff
@@ -22,6 +22,7 @@
     "noFallthroughCasesInSwitch": true,
     "forceConsistentCasingInFileNames": true,
     "importsNotUsedAsValues": "error",
+    "ignoreDeprecations": "5.0",
```

Comment:
Silencing the deprecation of `importsNotUsedAsValues`

Reply:
I hope you will remember why this line (and the `sourceMap: false` from `cypress` ) is here cause I will not :joy:

Reply:
Hehe :joy:  I guess this line is easy: you remove it and you get a warning in the IDE. The other warning shows up when running Cypress, so it's less obvious.